acceptance/config/beaker/options.rb

```diff
@@ -8,6 +8,7 @@
   "puppetservice" => 'puppetserver',
   "use-service" => true,
   "master-start-curl-retries" => 60,
-  "puppetpath" => "/etc/puppetlabs/agent"
+  "puppetpath" => "/etc/puppetlabs/puppet",
+  "puppetserver-confdir" => '/etc/puppetlabs/puppetserver/conf.d'
```

Comment:
puppet (ruby) no longer uses `puppetpath` and that option was removed from the AIO host objects in beaker. Ideally you should be able to remove `puppetpath` as well (now that you've moved the pin to dc53fa36a1813454b9b55243a3f291dae122d614), but perhaps there are still puppetserver acceptance tests that rely on that seting?